cob_obstacle_distance/include/cob_obstacle_distance/marker_shapes.hpp

```diff
@@ -52,30 +51,30 @@ class MarkerShape : public IMarkerShape
         FclMarkerConverter<T> fcl_marker_converter_;
         visualization_msgs::Marker marker_;
 
-        void init(double x, double y, double z,
+        void init(const std::string root_frame, double x, double y, double z,
                   double quat_x, double quat_y, double quat_z, double quat_w,
                   double color_r, double color_g, double color_b, double color_a);
 
     public:
-        MarkerShape(T &fcl_object, geometry_msgs::Pose &pose, std_msgs::ColorRGBA &col)
-        : MarkerShape(fcl_object,
+        MarkerShape(const std::string root_frame, T& fcl_object, geometry_msgs::Pose& pose, std_msgs::ColorRGBA& col)
+        : MarkerShape(root_frame, fcl_object,
```

Comment:
see this [comment](https://github.com/ipa-fxm/cob_control/pull/7/files#r34412108)